sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/DecryptParameters.cs

```diff
@@ -105,6 +105,16 @@ public static DecryptParameters A192CbcParameters(byte[] ciphertext, byte[] iv)
         public static DecryptParameters A256CbcParameters(byte[] ciphertext, byte[] iv) =>
             new DecryptParameters(EncryptionAlgorithm.A256Cbc, ciphertext, iv);
 
+        /// <summary>
+        /// Creates an instance of the <see cref="DecryptParameters"/> class for the <see cref="EncryptionAlgorithm.CkmAesKeyWrap"/> encryption algorithm.
+        /// </summary>
+        /// <param name="ciphertext">The ciphertext to decrypt.</param>
```

Comment:
```suggestion
        /// <param name="ciphertext">The cipher text to decrypt.</param>
```

Leaving one comment, but we should add a space to all of the instances in the docs.